skillmap/src/components/makecodeFrame.tsx

```diff
@@ -165,6 +163,10 @@ class MakeCodeFrameImpl extends React.Component<MakeCodeFrameProps, MakeCodeFram
 
         switch (data.action) {
             case "newproject":
+                if (!this.state.workspaceReady) {
+                    this.setState({ workspaceReady: true });
```

Comment:
>this.setState({ workspaceReady: true }); [](http://example.com/codeflow?start=20&length=40)

Will queued messages be sent at this point?

Reply:
oops, clearing the queue got dropped while i was cleaning stuff up! added it back